tools/Vis_PV.py

```diff
@@ -0,0 +1,286 @@
+        clip1.ClipType = 'Sphere'
+        clip1.Scalars = ['POINTS', var]
+        clip1.ClipType.Center = clip_Properties['Center_s']
+        clip1.ClipType.Radius = clip_Properties['Radius_s']
+    elif clip_Properties['Clip_type'] == 'Cylinder':
+        clip1.ClipType = 'Cylinder'
+        clip1.Scalars = ['POINTS', var]
+        clip1.ClipType.Center = clip_Properties['Center_c']
+        clip1.ClipType.Radius = clip_Properties['Radius_c']
+        clip1.ClipType.Axis = clip_Properties['Axis']
+    #--Hide previous data display before filter--
+    Hide(Filter1, renderView1)
+    xDMFReader1_Display = Show(clip1,renderView1)
+    return renderView1, clip1, xDMFReader1_Display
+
+def apply_slice(slice_Properties, var, renderView1, Filter1):
+    slice1 = Slice(Input=Filter1)
+    if slice_Properties['Slice_type'] == 'Plane':
+        slice1.SliceType = 'Plane'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Origin= slice_Properties['Origin']
+        slice1.SliceType.Normal= slice_Properties['Normal']
+    elif slice_Properties['Slice_type'] == 'Box':
+        slice1.SliceType = 'Box'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Position= slice_Properties['Position']
+        slice1.SliceType.Rotation= slice_Properties['Rotation']
+        slice1.SliceType.Scale= slice_Properties['Scale']
+    elif slice_Properties['Slice_type'] == 'Sphere':
+        slice1.SliceType = 'Sphere'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Center = slice_Properties['Center_s']
+        slice1.SliceType.Radius = slice_Properties['Radius_s']
+    elif slice_Properties['Slice_type'] == 'Cylinder':
+        slice1.SliceType = 'Cylinder'
+        slice1.SliceOffsetValues = [0.0]
+        slice1.SliceType.Center = slice_Properties['Center_c']
+        slice1.SliceType.Radius = slice_Properties['Radius_c']
+        slice1.SliceType.Axis = slice_Properties['Axis']
+    #--Hide previous data display before filter--
+    Hide(Filter1, renderView1)
+    xDMFReader1_Display = Show(slice1,renderView1)
+    return renderView1, slice1, xDMFReader1_Display
+
+def apply_warp(var, xDMFReader1, renderView1):
+    '''
+    Add
+    '''
+    slice1 = Slice(Input=xDMFReader1)
+    slice1.SliceType = 'Plane'
+    slice1.SliceOffsetValues = [0.0]
+    slice1.SliceType.Origin = [3.141592653589793, 3.141592653589793, 3.141592653589793]
+    slice1.SliceType.Normal = [0.0, 0.0, 1.0]
+    #xDMFReader1_Display.ColorArrayName = ['POINTS', var]
+    warpByScalar1 = WarpByScalar(Input=slice1)
+    warpByScalar1.Scalars = ['POINTS', var]
+    Hide(xDMFReader1, renderView1)
+    xDMFReader1_Display = Show(warpByScalar1, renderView1)
+    return renderView1
+
+def save_images(renderView1, xDMFReader1):
+    '''
+    Saves single image or multiple images based on number of 
+    time steps in data. One image per time step saved
+    '''
+    time_steps = xDMFReader1.TimestepValues
+    renderView1.ViewSize = [1330, 670]
```

Comment:
`[1920, 1080]`, and maybe an option to do 720p and 480p also so that people can reduce render time.